gtsfm/bundle/bundle_adjustment.py

```diff
@@ -90,11 +86,18 @@ def __init__(
             shared_calib (optional): Flag to enable shared calibration across all cameras. Defaults to False.
             max_iterations (optional): Max number of iterations when optimizing the factor graph. None means no cap.
                 Defaults to None.
+            cam_pose3_prior_noise_sigma (optional): Camera pose3 prior noise sigma. Default to 0.1.
```

Comment:
nit capitalization: "Camera Pose3 prior noise sigma." Can omit what it defaults to, since redundant with function signature, and comment may go stale quickly.

Reply:
Resolved, thanks!